metrics/influx.go

```diff
@@ -0,0 +1,165 @@
+package metrics
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/prebid/prebid-cache/config"
+	"github.com/rcrowley/go-metrics"
+	"github.com/vrischmann/go-metrics-influxdb"
+)
+
+type MetricsEntry struct {
```

Comment:
Another way of viewing how this MetricsEntry might look for Prometheus is with...
 - Duration Histogram (non-vectorized). That would use 13 counters if you specify 10 buckets.
 - Request Counter (vectorized) with a status label of values 'ok', 'badRequest', and 'error'. I think that would use 3 counters.

Reply:
Ok. Just to clarify:
```
 35 type PrometheusMetricsEntry struct {
 36   - Duration   *prometheus.HistogramVec
 37   - Errors     prometheus.Counter
 38   - BadRequest prometheus.Counter
 39   - Request    prometheus.Counter
 40
 41   + Duration   prometheus.Histogram  //Non vector
 42   + RequestStatus *prometheus.CounterVec  // CounterVec "status": "ok", "error", or "bad_request"
 43   + /*initialized like this:
 44   + metrics.requests = newCounter(cfg, metrics.Registry,
 45   +     "requests",
 46   +     "Count of total requests to Prebid Server labeled by status.",
 47   +     []string{"status"})
 48   + */
 49 }
metrics/prometheus.go [+]
```
Right?

Reply:
Yup. Maybe name it simply `Requests`.